Common/include/CConfig.hpp

```diff
@@ -3366,6 +3376,13 @@ class CConfig {
    */
   void SetMarker_All_Deform_Mesh(unsigned short val_marker, unsigned short val_deform) { Marker_All_Deform_Mesh[val_marker] = val_deform; }
 
+  /*!
+   * \brief Set if a marker <i>val_marker</i> allows deformation at the boundary.
+   * \param[in] val_marker - Index of the marker in which we are interested.
+   * \param[in] val_interface - 0 or 1 depending if the the marker is or not a MATCH_DEFORM_MESH marker.
+   */
```

Comment:
I like the idea for the "sliding" mesh deformation at some walls but the name is probably not very intuitive, maybe MARKER_DEFORM_MESH_SYM_PLANE?

I would also reduce the methods added to CConfig to the bare minimum as it is already a gigantic class, for example `GetnMarker_Match_Deform_Mesh` does not seem to be used.

Reply:
Oh! And we should prevent markers from being both DEFORM_MESH and DEFORM_MESH_SYM_PLANE